lib/annotations/ratios.js

```diff
@@ -1,11 +1,43 @@
 'use strict';
 
+const ensureSassJson = require('../utils/ensureSassJson');
+const renderIframe = require('../renderIframe');
+
 /**
  * Custom `@ratios` annotation. Accepts optional
  * map-variable name (used to access data from Sass JSON).
  */
-module.exports = () => ({
+module.exports = env => ({
```

Comment:
🤦🏾 `ratios` somehow slipped by me; thinking this affected just `fonts` and `sizes`.  
(It's not clear to me the difference between `sizes` and `ratios`)

Reply:
Yeah, I forgot those were distinct annotations (though very similar) also.

It definitely took me a bit to get my mind back around how this all works... The annotation (e.g. `/// @ratios`) from scss is parsed and turned into a data object, then that is passed into the `resolve` fn which can add more data (e.g. rendering markup in an iframe), and then all of that is passed through to the base nunjucks templates...